src/test/java/org/jabref/logic/search/query/SearchQueryLuceneConversionTest.java

```diff
@@ -10,64 +12,55 @@
 
 class SearchQueryLuceneConversionTest {
 
-    public static Stream<Arguments> transformationNormal() {
+    public static Stream<Arguments> testSearchConversion() {
         return Stream.of(
-                // If "any:" should not be added, see e46e0a23d7b9526bd449cbecfb189ae6dbc40a28 for a fix
-                Arguments.of("any:chocolate", "chocolate"),
+                Arguments.of("content:term annotations:term", "term"),
+                Arguments.of("content:term annotations:term", "any = term"),
+                Arguments.of("content:term annotations:term", "any CONTAINS term"),
+                Arguments.of("content:term annotations:term", "any MATCHES term"),
+                Arguments.of("content:term annotations:term", "any =! term"),
+                Arguments.of("content:term annotations:term", "any == term"),
+                Arguments.of("content:term annotations:term", "any ==! term"),
 
-                Arguments.of("title:chocolate", "title=chocolate"),
```

Comment:
I am better in reading concrete names. I somehow miss these basic tests in the new code. (image proceccsing below as kept, but not these ones)